apps/aehttp/src/aehttp_dispatch_ext.erl

```diff
@@ -312,7 +321,9 @@ handle_request('PostSpend', #{'SpendTx' := Req}, _Context) ->
     ParseFuns = [parse_map_to_atom_keys(),
                  read_required_params([sender,
                                        {recipient_pubkey, recipient},
-                                       amount, fee, ttl, payload]),
+                                        amount, fee, payload]),
+                 read_optional_params([{ttl, ttl, '$no_value'}]),
+                 read_optional_params([{ttl, ttl, '$no_value'}]),
```

Comment:
Twice?

Reply:
No, rebasing gone wrong. But no harm in doing it twice ;-)

Reply:
Will fix of course